static/explorer.scss

```diff
@@ -24,6 +24,15 @@
     cursor: pointer;
 }
 
+
```

Comment:
nit: please replace this extra blank line with a comment about the features yet to land in monaco: https://github.com/microsoft/vscode/issues/16038